src/tests/relay/t_relay_io_handle_sidre.cpp

```diff
@@ -48,6 +48,7 @@ TEST(conduit_relay_io_handle, test_sidre_basic)
     // create an equiv conduit tree for testing
     //
 
+    conduit::int64    conduit_vals_0[]  = {};
```

Comment:
Appveyor was not happy with this line:
```
  c:\projects\conduit\src\tests\relay\t_relay_io_handle_sidre.cpp(51): error C2466: cannot allocate an array of constant size 0 [C:\projects\conduit\build\tests\relay\t_relay_io_handle_sidre.vcxproj]
```
See: https://ci.appveyor.com/project/cyrush/conduit/builds/38332028/job/9u4dj5d7edmp7d6a#L3935